File: BaekJoon/Bronze/2920/Python/2920.py
import sys

def getStatus(array):
    if array[0] == 1:
        for i in range(0, len(array)):
            if i+1 != array[i]:
                return 'mixed'
        return 'ascending'
    elif array[0] == 8:
        for i in range(0, len(array)):
            if len(array)-i != array[i]:
                return 'mixed'
        return 'descending'
    else:
        return 'mixed'

arr1 = list(map(int, sys.stdin.readline().split()))
print(getStatus(arr1))

File: BaekJoon/Bronze/2920/C++/2920.cpp
// no.2920: 음계 (B2)

#include <cstdio>
#include <sstream>
using namespace std;

int main() {
    char str[100];
    fgets(str, 100, stdin);
    stringstream ss(str);
    string curLetter;
    ss >> curLetter;
    if(curLetter == "1") {
        int i=2;
        while(ss >> curLetter) {
            int curNum = stoi(curLetter);
            if(curNum != i++) {
                printf("mixed\n");
                return 0;
            }
        }
        printf("ascending\n");
    } else {
        if(curLetter != "8") {
            printf("mixed\n");
            return 0;
        }
        else {
            int i=7;
            while(ss >> curLetter) {
                int curNum = stoi(curLetter);
                if(curNum != i--) {
                    printf("mixed\n");
                    return 0;
                }
            }
        }
        printf("descending\n");
    }
    return 0;
}